pkg/scheduler/framework/interface_test.go

```diff
@@ -120,3 +121,69 @@ func TestPluginToStatusMerge(t *testing.T) {
 		}
 	}
 }
+
+var (
+	successStatus   = NewStatus(Success)
+	errorStatus     = NewStatus(Error, "internal error")
+	unstableStatusX = NewStatus(Unschedulable, "unstable")
+	unstableStatusY = NewStatus(Unschedulable, "unstable")
```

Comment:
unstable or unschedulable?

nit: It's favorable to define Status in the testcases instead of refer to private variables.